ocp4crd/oauth/oauth_test.go

```diff
@@ -0,0 +1,48 @@
+package oauth
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	ocp3 "github.com/fusor/cpma/ocp3config"
+)
+
+func TestGenerateMasterConfig(t *testing.T) {
+	testConfig := ocp3.Config{
+		Masterf: "../../examples/source/etc/origin/master/master-config.yaml",
+	}
+	masterConfig := testConfig.ParseMaster()
+
```

Comment:
Defining identity providers manually took some time, but I would like to be 146% sure our generated YML is valid.